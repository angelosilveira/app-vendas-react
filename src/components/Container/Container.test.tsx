import { screen } from '@testing-library/react';
import { renderWithTheme } from 'src/utils/tests/renderWithTheme';
import { describe, test, expect } from 'vitest';

import { Container } from '.';

describe('<Container />', () => {
  test('should render the container', () => {
    const { container } = renderWithTheme(
      <Container>
        <span>App vendas</span>
      </Container>
    );

    expect(screen.getByText(/app vendas/i)).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
