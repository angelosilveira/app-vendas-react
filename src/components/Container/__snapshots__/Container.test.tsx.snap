// Vitest Snapshot v1

exports[`<Container /> > should render the container 1`] = `
.c0 {
  padding-left: 15px;
  padding-right: 15px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100%;
}

<div
  class="c0"
>
  <span>
    App vendas
  </span>
</div>
`;
